buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -163,8 +186,30 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
+      "displayName": "symbol packages -> Azure Feed",
+      "timeoutInMinutes": 0,
+      "condition": "and(succeeded(), eq(variables['PublishToAzureFeed'], 'True'))",
+      "task": {
+        "id": "e213ff0f-5d5c-4791-802d-52ea3e7be1f1",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "scriptType": "inlineScript",
+        "scriptName": "",
+        "arguments": "-ConfigurationGroup $(PB_ConfigurationGroup) -OutputFeedUrl $(PB_AzureFeedUrl) -StorageAccountKey $(PB_PublishAccountKey)",
+        "workingFolder": "$(Pipeline.SourcesDirectory)",
+        "inlineScript": "param($ConfigurationGroup, $OutputFeedUrl, $StorageAccountKey)\nif ($ConfigurationGroup -ne \"Release\") { exit }\n\n $scriptpath = $MyInvocation.MyCommand.Path \n $dir = Split-Path $scriptpath \n msbuild src\\publish.proj /t:FinalizePublish /p:AccountKey=$StorageAccountKey /p:ExpectedFeedUrl=$OutputFeedUrl /p:ConfigurationGroup=$ConfigurationGroup /p:PublishSymbols=\"true\"",
+        "failOnStandardError": "true"
+      }
+    },
+    {
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
       "displayName": "Update versions repository",
       "timeoutInMinutes": 0,
+      "condition": "and(succeeded(), eq(variables['PublishToMyGet'], 'True'))",
```

Comment:
In a world where we use Azure Feeds, do we also want to update the versions repo when that is used?